doctest.cabal

```diff
@@ -1,10 +1,8 @@
-cabal-version: 1.12
+cabal-version: 2.0
 
 -- This file has been generated from package.yaml by hpack version 0.34.4.
 --
 -- see: https://github.com/sol/hpack
---
--- hash: d72341386fa5019d836d102ce53149841ad11aa7bd786c3ba9b8cc28ca3c9046
 
```

Comment:
Does this mean that `doctest.cabal` is no longer created by `hpack`?

Reply:
No, it's still created by `hpack`.  It merely means that `hpack` will overwrite the file, even if it was modified manually.